3rd_Homework/3.1/pages/index.css

```diff
@@ -0,0 +1,1396 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding-bottom: 32px;
+    position: relative;
+    background: #F2F4F7;
+}
+
+/*
+    sidebar CSS HERE
+*/
+.sidebar{
+    position: relative;
+    box-sizing: border-box;
+    width: 16.11%;
+    height: 1024px;

```

Comment:
we can't be sure that a user has device height equals 1024px. and there is better to use 100vh. 

actually, if my screen if smaller then 1024px, this sidebar will be scrollable, but i am not sure that this is an expected behaviour


https://user-images.githubusercontent.com/127765924/228969959-dbf2a312-6493-4516-9648-c0e4c5e13fb4.mov

Reply:
and it is better to make the sidebar is sticky :)